src/Catty/DataModel/Bricks/Look/SetBackgroundBrick.m

```diff
@@ -78,21 +78,4 @@ - (NSInteger)getRequiredResources
     return kNoResources;
 }
 
-
-- (id)mutableCopyWithContext:(CBMutableCopyContext*)context
```

Comment:
Different to Bricks containing Formulas, Looks should not be copied when a Brick gets copied. Looks are Object-wide assets. So a copied _SetBackgroundBrick_ within an Object should still point to the same Look within that Object. Exception: when copying the whole Object, this reference needs to be updated. Therefore I think we still need that method -- probably some of the _SpriteObjectMutableCopyTests_ will be failing. Please add a separate test to your _SetBackgroundBrickTests_ checking this behavior by making use of the _CBMutableCopyContext_.

Reply:
Got the point! Upon adding the previously removed method, I think I should also add another override of that method which have reportError bool argument. Otherwise, those calls will be redirected to parent method.

Reply:
Thanks! And good point. The interface of the Brick is misleading since the other method could be called as well, right. However, this may cause a few more changes in this pr which makes review even more difficult, so in this ticket please only stick to the _mutableCopyWithContext_ method like the one you deleted here. I will create a separate ticket for the concern you mentioned which will deal with this issue.